src/Forms/FormInput.js

```diff
@@ -24,8 +31,13 @@ FormInput.displayName = 'FormInput';
 FormInput.propTypes = {
     className: PropTypes.string,
     disabled: PropTypes.bool,
+    id: PropTypes.string,
+    name: PropTypes.string,
+    placeholder: PropTypes.string,
     readOnly: PropTypes.bool,
-    state: PropTypes.oneOf(INPUT_TYPES)
+    state: PropTypes.oneOf(INPUT_TYPES),
+    type: PropTypes.string,
+    value: PropTypes.string
 };
 
```

Comment:
prop description should be provided for all non default props seen here:

https://github.com/SAP/fundamental-react/blob/master/src/_playground/documentation/Properties/defaults.js